src/actions_ls.rs

```diff
@@ -212,7 +228,7 @@ impl ActionHandler {
         let vfs: &Vfs = &self.vfs;
         let result: Vec<CompletionItem> = panic::catch_unwind(move || {
             let pos = adjust_vscode_pos_for_racer(params.position);
-            let file_path = &params.textDocument.file_name();
+            let file_path = &uri_string_to_file_name(&params.text_document.uri);
```

Comment:
> Alternatively have a FileUri newtype that impls Deserialize to convert a string to a PathBuf via Url::to_file_path ?

The resource does not necessarily need to be a `file://` URI so limiting it to files in the type would be limiting.